### PR TITLE
Add noarch: python

### PR DESCRIPTION
**Pull request by williamjamir**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ source:
   {{ hash_type }}: {{ hash }}
 
 build:
+  noarch: python
   number: 1
```
Comment (jakirkham) [inline recipe/meta.yaml]: Please bump the build number as well.

**Comment by jakirkham** [comment]
Thanks @williamjamir. Do you have any interest in being a maintainer of this feedstock?

**Comment by williamjamir** [comment]
Hi @jakirkham, 
I have already added myself as a maintainer in PR #7, I appreciate your attention =)

File: recipe/meta.yaml (hunk 1 above)
Comment (williamjamir) [reply recipe/meta.yaml]: My last commit fixes this issue, sorry about that

**Comment by jakirkham** [comment]
LGTM.

**Pull request merged**